doctr/models/detection/differentiable_binarization/base.py

```diff
@@ -224,7 +226,7 @@ def draw_thresh_map(
         padded_polygon: np.ndarray = np.array(padding.Execute(distance)[0])
 
         # Fill the mask with 1 on the new padded polygon
-        cv2.fillPoly(mask, [padded_polygon.astype(np.int32)], 1.0)
+        cv2.fillPoly(mask.copy(), [padded_polygon.astype(np.int32)], 1.0)
```

Comment:
Will it really fill `mask` or will it fill another object with the same content as `mask` ?

Reply:
oh this shouldn't have been added removed it.